site/nav.yml

```diff
@@ -23,23 +23,24 @@ nav:
   - Docs: 
     - nightly: '!include docs/docs/nightly/mkdocs.yml'
     - latest: '!include docs/docs/latest/mkdocs.yml'
-    - 1.9.1: '!include docs/docs/1.9.1/mkdocs.yml'
-    - 1.9.0: '!include docs/docs/1.9.0/mkdocs.yml'
-    - 1.8.1: '!include docs/docs/1.8.1/mkdocs.yml'
-    - 1.8.0: '!include docs/docs/1.8.0/mkdocs.yml'
-    - 1.7.2: '!include docs/docs/1.7.2/mkdocs.yml'
-    - 1.7.1: '!include docs/docs/1.7.1/mkdocs.yml'
-    - 1.7.0: '!include docs/docs/1.7.0/mkdocs.yml'
-    - 1.6.1: '!include docs/docs/1.6.1/mkdocs.yml'
-    - 1.6.0: '!include docs/docs/1.6.0/mkdocs.yml'
-    - 1.5.2: '!include docs/docs/1.5.2/mkdocs.yml'
-    - 1.5.1: '!include docs/docs/1.5.1/mkdocs.yml'   
-    - 1.5.0: '!include docs/docs/1.5.0/mkdocs.yml'
-    - 1.4.3: '!include docs/docs/1.4.3/mkdocs.yml'
-    - 1.4.2: '!include docs/docs/1.4.2/mkdocs.yml'
-    - 1.4.1: '!include docs/docs/1.4.1/mkdocs.yml'
-    - 1.4.0: '!include docs/docs/1.4.0/mkdocs.yml'
-    - archive: archive.md
+    - Previous:
+      - 1.9.1: '!include docs/docs/1.9.1/mkdocs.yml'
```

Comment:
1.9.1 is the latest version. This is why I proposed https://github.com/apache/iceberg/issues/12917